manifests/m/Microsoft/PowerShell/7.3.2.0/Microsoft.PowerShell.installer.yaml

```diff
@@ -0,0 +1,44 @@
+# Created with YamlCreate.ps1 v2.2.2 $debug=NVS1.CRLF.7-3-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Microsoft.PowerShell
+PackageVersion: 7.3.2.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Commands:
+- pwsh
+ReleaseDate: 2023-01-24
+Installers:
+- Architecture: x64
+  InstallerType: wix
+  InstallerUrl: https://github.com/PowerShell/PowerShell/releases/download/v7.3.2/PowerShell-7.3.2-win-x64.msi
+  InstallerSha256: A4F7D081C5F74BC8D6C75F1DFEE382B7FD9335361181748FEE590ECDBC96CB26
+  ProductCode: '{323AD147-6FC4-40CB-A810-2AADF26D868A}'
+- Architecture: x86
+  InstallerType: wix
+  InstallerUrl: https://github.com/PowerShell/PowerShell/releases/download/v7.3.2/PowerShell-7.3.2-win-x86.msi
+  InstallerSha256: 43EBD047A0AC8951D01C59E8F42F27A50DA37556D779E99B4AFF009EF8418B5E
+  ProductCode: '{EE9CC6F4-ACF6-4B10-81BD-49D1C188770D}'
+- Architecture: arm
+  MinimumOSVersion: 10.0.17763.0
+  Platform:
+  - Windows.Universal
+  InstallerType: msix
+  InstallerUrl: https://github.com/PowerShell/PowerShell/releases/download/v7.3.2/PowerShell-7.3.2-win.msixbundle
```

Comment:
The MSIXbundle packages are only intended for the MSStore source; Please update the PR to only include the msi packages

Reply:
Our team (PowerShell) is working to support msi for arm, but until then the temporary solution is to include the msix installer only for arm so this is needed in the installer manifest for now.

Reply:
@trenly Our team (PowerShell) is working to support msi for arm, but until then the temporary solution is to include the msix installer only for arm so this is needed in the installer manifest for now.

Reply:
Ah, I see. I was only aware that it had caused issues in the past